schoolUI/schoolUI/ViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ViewController.swift
+//  Lesson 7 homework
+//
+//  Created by Ernest Mihasenko on 22.03.22.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    var studentsArray = [Student]()
+ 
+    @IBOutlet weak var studentSurnameTextField: UITextField!
+    @IBOutlet weak var studentNameTextField: UITextField!
+    @IBOutlet weak var addStudentButton: UIButton!
+    @IBOutlet weak var studentGradeTextField: UITextField!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+
+    }
+    
+
+
+    
+    @IBAction func addStudent(_ sender: Any) {
+        let school = School(students: studentsArray)
```

Comment:
It's better to create school as a property of `ViewContoller`: `let school = School(students: [])` and then add student like this: `school.students.append(student)`